clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3922,16 +3922,41 @@ def LifetimeCaptureByDocs : Documentation {
   let Category = DocCatFunction;
   let Content = [{
     Similar to `lifetimebound`_, the ``lifetime_capture_by(X)`` attribute on a function
-parameter or implicit object parameter indicates that that objects that are referred to
-by that parameter may also be referred to by the capturing entity ``X``.
+parameter or implicit object parameter indicates that the capturing entity
+``X`` may refer to the object referred by that parameter.
```

Comment:
nit: You can probably move some text to the previous line.

Reply:
The ``X``  didn't fit in 80 lines. Reformatted the para.